apps/app/platform.tf

```diff
@@ -71,6 +76,130 @@ resource "azurerm_linux_web_app" "app" {
       }
     }
   }
+
+  lifecycle {
+    ignore_changes = [
+      auth_settings_v2
+    ]
+  }
+}
+
+# workaround for https://github.com/hashicorp/terraform-provider-azurerm/issues/20820
```

Comment:
yeah... I'm not sure it'd look much nicer because of either the dynamic blocks or copy-paste between webapp/slot resource. happy to go with whatever you think though 🤷🏼